File: JS/10-18-25/Java/PerfectNumber.java
public class PerfectNumber{
public static void main(String [] args){
	int number = 7;
	int sum = 0;


	for( int divisor = 1; divisor < number; divisor++ ){
		if( number % divisor == 0 )
			sum += divisor;
		}

	if( sum == number )
		System.out.printf("%d is a positive number", number);
	else
		System.out.printf("%d is not positive number", number);

}}

File: JS/10-18-25/JS/perfect_number.js
let number = 7;
let sum = 0

for (let divisor = 1;divisor < number; divisor++){
	if (number % divisor == 0)
		sum += divisor;

if (sum == number)
	console.log("It is a perfect number");
else
	console.log("It is not a perfect number");
